src/http_server.cc

```diff
@@ -243,8 +243,6 @@ TRITONSERVER_Error*
 JsonBytesArrayByteSize(
```

Comment:
Can you add a short description of this function?
Because the name doesn't do the justice as it can recursively accumulate the byte size.